app/src/main/res/values/strings.xml

```diff
@@ -59,12 +60,28 @@
     <string name="date_unknown">Datum unbekannt</string>
 
     <!-- Tourbook -->
+    <string name="ascends">Begehungen</string>
     <string name="botch">\u0020\u26A1</string>
     <string name="botch_text_symbol">Säcke \u26A1</string>
     <string name="watching">\u0020\u261D</string>
     <string name="project">\u0020\u2117</string>
     <string name="project_text_symbol">Projekte \u2117</string>
 
+    <!-- File IO -->
+    <string name="tourbook_filename">Tourenbuch.json</string>
```

Comment:
I think this should be stored in the resource file `setup.xml` because it is not a string that will be translated.